test/event_test.rb

```diff
@@ -0,0 +1,75 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/event.rb'
+
+class EventTest < Minitest::Test
```

Comment:
This test suite looks GREAT!  You are testing every method and the names and syntax look awesome.